src/lib/progress/steps/horizontal-bullet.tsx

```
@@ -93,14 +104,26 @@
+    if (textRef.current) {
+      const totalHeight = textRef.current.clientHeight;
+      setPaddingBottom(totalHeight);
+    }
+  }, [subitems]);
+
+  return (
+    <Container {...{ active, completed, last, paddingBottom }} {...props}>
+      <BulletCircle {...{ active, completed, index }} />
+      <TextWrapper ref={textRef} {...{ active }}>
+        <h2>{title}</h2>
+        {subitems && subitems.map((item, i) => <small key={i}>{item}</small>)}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)